Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/Content-Management-Sample/index.md

```diff
@@ -0,0 +1,280 @@
+    var updatedContent = await _contentManagementService.Content.Update(yourContentObject);
+    ```
+
+    :::note
+    You can create content programmatically by following the [Create content programmatically](../../../../Reference/Management/Services/ContentService/create-content-programmatically.md) guide.
+    :::
+
+### Publish Content
+
+The `ContentManagementService` has one method for publishing content.
+
+1. `Publish(Guid id, string culture = "*")` - Publishes an existing content item based on the passed-in GUID and culture.
+
+    To use this method, call the method on the `ContentManagementService` instance and pass in the GUID of the content item you want to publish:
+
+    ```csharp
+    var publishedContent = await _contentManagementService.Content.Publish(yourGuidObject);
+    //or
+    var publishedContent = await _contentManagementService.Content.Publish(Guid.Parse("your-guid-as-a-string"));
+    ```
+
+    Optionally, you can also pass in the culture of the content item you want to publish:
+
+    ```csharp
+    var publishedContent = await _contentManagementService.Content.Publish(yourGuidObject, "en-US");
+    //or
+    var publishedContent = await _contentManagementService.Content.Publish(Guid.Parse("your-guid-as-a-string"), "en-US");
+    ```
+
+### Unpublish Content
+
+The `ContentManagementService` has one method for unpublishing content.
+
+1. `Unpublish(Guid id, string culture = "*")` - Unpublishes an existing content item based on the passed-in GUID and culture.
+
+    To use this method, call the method on the `ContentManagementService` instance and pass in the GUID of the content item you want to unpublish:
+
+    ```csharp
+    var unpublishedContent = await _contentManagementService.Content.Unpublish(yourGuidObject);
+    //or
+    var unpublishedContent = await _contentManagementService.Content.Unpublish(Guid.Parse("your-guid-as-a-string"));
+    ```
+
+    Optionally, you can also pass in the culture you want to unpublish the content item in:
+
+    ```csharp
+    var unpublishedContent = await _contentManagementService.Content.Unpublish(yourGuidObject, "en-US");
+    //or
+    var unpublishedContent = await _contentManagementService.Content.Unpublish(Guid.Parse("your-guid-as-a-string"), "en-US");
+    ```
+
+### Delete Content
+
+The `ContentManagementService` has one method for deleting content.
+
+1. `Delete(Guid id)` - Deletes an existing content item based on the passed-in GUID.
+
+    To use this method, call the method on the `ContentManagementService` instance and pass in the GUID of the content item you want to delete:
+
+    ```csharp
+    var deletedContent = await _contentManagementService.Content.Delete(yourGuidObject);
+    //or
+    var deletedContent = await _contentManagementService.Content.Delete(Guid.Parse("your-guid-as-a-string"));
+    ```
+
+## Media
+
+First, we will look at how to work with media. The `ContentManagementService` allows you to Get, Create, Update and Delete media.
```

Comment:
Content was "first", so I guess Media also can't be "first"? 😄 

I think, let's take out the "First"s all together. I don't people will be reading an article like this from top to bottom anyways, right?
What do you think?

Reply:
I agree. I reworded this to align with your thoughts. What do you think? :smiley: